install.hs

```diff
@@ -534,23 +544,23 @@ stackBuildFailMsg =
     ++ "\thttps://github.com/haskell/haskell-ide-engine"
 
 -- |No suitable ghc version has been found. Show a message.
-ghcVersionNotFound :: VersionNumber -> String
-ghcVersionNotFound versionNumber =
+ghcVersionNotFoundFailMsg :: VersionNumber -> String
+ghcVersionNotFoundFailMsg versionNumber =
   "No GHC with version "
     <> versionNumber
     <> " has been found.\n"
     <> "Either install a fitting GHC, use the stack targets or modify the PATH variable accordingly."
 
 -- | Error message when a windows system tries to install HIE via `cabal new-install`
-cabalInstallNotSuported :: String
-cabalInstallNotSuported =
+cabalInstallNotSuportedFailMsg :: String
+cabalInstallNotSuportedFailMsg =
   "This system has been identified as a windows system.\n"
     ++ "Unfortunately, `cabal new-install` is currently not supported on windows.\n"
     ++ "Please use one of the stack-based targets.\n\n"
     ++ "If this system has been falsely identified, please open an issue at:\n\thttps://github.com/haskell/haskell-ide-engine\n"
 
 -- | Error message when a windows system tries to install HIE via `cabal new-install`
```

Comment:
Wrong documentation

Reply:
What is the problem?

Reply:
Documentation is duplicated for the function `stackExeIsOldFailMsg`.